source/stylesheets/pages/_home.scss

```diff
@@ -18,7 +18,10 @@ section.intro {
   br {
     display: block;
   }
-  p {
+  button {
+    appearance: none;
+    background: transparent;
+    border: none;
```

Comment:
I applied some general clean-up styles to the button in order to hide the browser's default view for it. You might want to refactor this into a utility class that you can apply globally when you want to remove all of the default button features.

Reply:
Will do!